include/abb_librws/rws_interface.h

```diff
@@ -64,19 +64,75 @@ class RWSInterface
      */
     std::string system_name;
   };
-  
+
   /**
-   * \brief A struct for containing information about the RAPID tasks defined in the robot controller.
+   * \brief A struct for containing information about a RobotWare option.
    */
-  struct RAPIDTask
+  struct RobotWareOptionInfo
+  {
+    /**
+     * \brief A constructor.
+     *
+     * \param name for the name of the option.
+     * \param description for the description of the option.
+     */
+    RobotWareOptionInfo(std::string name, std::string description)
+    :
+    name(name),
+    description(description)
+    {}
+
+    /**
+     * \brief The option's name.
+     */
+    std::string name;
```

Comment:
Please use `_` suffixes on member variables.

```suggestion
    std::string name_;
```

Reply:
I have used the `_` suffix for private member variables only, and for `struct` data members I have mainly followed the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html).